sympy/plotting/plot.py

```diff
@@ -1551,8 +1554,30 @@ def plot(*args, **kwargs):
     show = kwargs.pop('show', True)
     series = []
     plot_expr = check_arguments(args, 1, 1)
-    series = [LineOver1DRangeSeries(*arg, **kwargs) for arg in plot_expr]
 
+    new_plot = []
+    for arg in plot_expr:
+        expr = arg[0]
+        limit = arg[1]
+        frac = fraction(expr) #to find the denominator
+        new_deno = frac[1]
+        if (not new_deno.is_Integer) and (not new_deno.is_Float) and (not new_deno.is_irrational):
```

Comment:
Why does it matter if they are irrational?

Reply:
Consider the following function `sin(x) / pi`, as in this function denominator is constant and was an irrational number. So for such situation, this was added.